docs/guides/use_checkpointing.md

```diff
@@ -23,45 +26,58 @@ This guide covers the following:
 * How to save and load checkpoints when you run in multi-host scenarios with
 [`flax.training.checkpoints.save_checkpoint_multiprocess`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#flax.training.checkpoints.save_checkpoint_multiprocess).
 
+
 ## Setup
 
 Install/upgrade Flax, JAX, [Optax](https://optax.readthedocs.io/) and [TensorStore](https://google.github.io/tensorstore/). For JAX installation with GPU/TPU support, visit [this section on GitHub](https://github.com/google/jax#installation).
 
-```{code-cell} ipython3
-!pip install -U -q flax jax jaxlib optax tensorstore
+```python
+!pip install -U -qq flax orbax
+
+# To use the latest checkpoint features, install Flax from head.
+# Note: This part will be removed/replaced after the next Flax release.
+!pip3 install --upgrade --no-deps --force-reinstall -qq \
+"git+https://github.com/google/flax.git@main#egg=flax"
 ```
 
 Note: Before running `import jax`, create eight fake devices to mimic multi-host checkpointing in this notebook. Note that the order of imports is important here. The `os.environ["XLA_FLAGS"] = '--xla_force_host_platform_device_count=8'` command works only with the CPU backend. This means it won't work with GPU/TPU acceleration on if you're running this notebook in Google Colab. If you are already running the code on multiple devices (for example, in a 4x2 TPU environment), you can skip running the next cell.
 
-```{code-cell} ipython3
+```python
 import os
 os.environ["XLA_FLAGS"] = '--xla_force_host_platform_device_count=8'
 ```
 
-```{code-cell} ipython3
+```python
 from typing import Optional, Any
 import shutil
 
 import numpy as np
 import jax
 from jax import random, numpy as jnp
-from jax.experimental import maps, PartitionSpec, pjit
-from jax.experimental.gda_serialization.serialization import GlobalAsyncCheckpointManager
 
 import flax
 from flax import linen as nn
 from flax.training import checkpoints, train_state
 from flax import struct, serialization
+import orbax.checkpoint as orbax
+
 import optax
 ```
 
+```python
```

Comment:
If possible, maybe we can suggest this step after `pip install flax` since this is still in the "Setup" stage of the guide.

For example:

```
!pip install -U -qq flax orbax
...

# Orbax requires asyncio to be insabled in a Google Colab environment
!pip install nest_asyncio
```

Then in the next code cell:

```
import jax
...
import optax
...
import nest_asyncio
nest_asyncio.apply()
```

LMKWYT